addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12284,7 +12284,18 @@ msgctxt "#20223"
 msgid "Artist information folder"
 msgstr ""
 
-#empty strings from id 20224 to 20239
+#: system/settings/settings.xml
+msgctxt "#20224"
+msgid "Prefer scraped album art"
+msgstr ""
+
+#. Description of setting with label #20224 "Prefer scraped album art"
+#: system/settings/settings.xml
+msgctxt "#20225"
+msgid "Downloaded album covers will be prefered over images extracted from embedded within a music file"
```

Comment:
extract from what ?

Reply:
"... extracted from embedded within a music file"
But this obviously sounds wrong to you @notspiff, and actually it doesn't sound perfect to me either so any other suggestions?

Reply:
```
msgid "Downloaded album covers will be prefered over embedded images extracted from within a music file" 
```
?

Reply:
Ah yes, I see it now, what was I on?
"Downloaded album covers will be preferred over those embedded within the music files"